CONTRIBUTING.md

```diff
@@ -146,11 +148,16 @@ summon ./test_e2e
    from your local machine).
 
 When releasing a new version of the Service Broker, you will need to include a
-ZIP file with the release of the repository with all dependencies. Running the
-`./bin/build` script will run `bundle pack --all`, which creates a
-`vendor/cache/` directory with the project dependencies. It will also produce a ZIP
-file of the project which includes this directory. The ZIP file should be uploaded
-to the release in GitHub; it will be used to build the VMWare Tanzu tile.
+ZIP file with the release of the repository with all dependencies.
+
+1. Verify that `dev/manifest.txt` includes all relevant top-level directories
+   and files.These will be copied into a temporary `pkg` directory used when
+   zipping, to avoid including unnecessary files in our ZIP.
+1. Run the `./dev/build` script, which will run `bundle pack --all`, which
```

Comment:
@BradleyBoutcher this reminds me, there is no "online buildpack" equivalent for the service broker, but this `bundle pack --all` command _does_ ensure the service broker can be installed in TAS foundations that are not internet-connected (eg so that the app can be installed / started without connecting to the internet to install deps).

I don't know if we test this functionality in this repo at all, but we would not want to break this.

Reply:
We run `bundle pack --all -no-install` before we perform the zipping process. Is that enough?

https://github.com/cyberark/conjur-service-broker/blob/ce243e1f68cc5a6784a90386f8cb087775a65c8a/dev/package#L15

Reply:
I think so. this is a hard one to test - but maybe you can try it if you download the artifact and start it with `./start-service-broker` with your internet turned off?